home/models.py

```diff
@@ -172,6 +172,15 @@ def __str__(self):
     def tags_feed(cls, search=''):
         return cls.objects.filter(tag_name__contains=search).order_by('tag_name')
 
+    @classmethod
```

Comment:
I think the test function for this method is missing.

Reply:
You right, I made a new file - `test_tag_model_functions.py` that will contain all of the tests that are related to his class models

Reply:
Thank you:)